Student_BCOs/Derek_Cao_BCO 000534.json

```diff
@@ -0,0 +1,168 @@
+                "status": "in-review",
+                "reviewer_comment": "Reviewed by BioCompute technical assistant, approval pending modification by author",
+                "reviewer": {
+                    "orcid": "https://orcid.org/0000-0002-8824-4637",
+                    "affiliation": "George Washington University",
+                    "contribution": [
+                        "curatedBy"
+                    ],
+                    "name": "Janisha Patel",
+                    "email": "janishapatel@gwu.edu"
+                },
+                "date": "10/16/2019"
+            }
+        ],
+        "contributors": [
+            {
+                "name": "Derek Cao",
+                "orcid": "https://orcid.org/0000-0002-7901-9470",
+                "affiliation": "George Washington University",
+                "contribution": [
+                    "createdBy"
+                ],
+                "email": "cderek7@gwu.edu"
+            },
+            {
+                "name": "Tiangong Lu",
+                "orcid": "",
+                "affiliation": "Department of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Armand Bankhead, III,",
+                "orcid": "",
+                "affiliation": "Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor ; Department of Biostatistics, School of Public Health, University of Michigan, Ann Arbor",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Mats Ljungman ",
+                "orcid": "https://orcid.org/0000-0003-1553-6695",
+                "affiliation": "Departments of Radiation Oncology, Rogel Cancer Center, University of Michigan Medical School and Department of Environmental Health Sciences, School of Public Health, University of Michigan, Ann Arbor",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Nouri Neamati",
+                "orcid": "https://orcid.org/0000-0003-3291-7131",
+                "affiliation": "Department of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            }
+        ]
+    },
+    "usability_domain": [
+        "Adapted from Tiangong Lu, Armand Bankhead, III,  Mats Ljungman, Nouri Neamati. \"Multi-omics profiling reveals key signaling pathways in ovarian cancer controlled by STAT3.\" PMC6735387(July 28, 2019). https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6735387/"
```

Comment:
While it's great that you included a sentence on which publication this BCO was adapted from, there also needs to be more information in this usability domain. See #122 for more information on what goes in here.